build.assets/tooling/cmd/build-apt-repos/s3manager.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gravitational/trace"
+	"github.com/seqsense/s3sync"
+	"github.com/sirupsen/logrus"
+)
+
+type s3manager struct {
+	syncManager *s3sync.Manager
```

Comment:
Why is `s3sync` needed? Doesn't AWS Go SDK support this out of the box?

Reply:
No - it is not supported on any of the AWS SDKs as far as I know.